.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
```

Comment:
I'll be honest, I don't really understand this file and on one of my computers F5 still worked fine without it. But I'm guessing this is the missing link for why it doesn't work on my other computers - thanks for adding.